praw/models/redditors.py

```diff
@@ -67,6 +75,36 @@ def stream(
 
         Keyword arguments are passed to :func:`.stream_generator`.
 
-        :returns Redditor profiles, which are a type of :class:`.Subreddit`.
+        :returns: Redditor profiles, which are a type of :class:`.Subreddit`.
         """
         return stream_generator(self.new, **stream_options)
+
+    def user_summaries(
```

Comment:
With the class name change, this would be `short_redditors` or something similar.